src/frontend/apps/lti_site/components/DashboardLiveJitsi/utils.ts

```diff
@@ -91,14 +91,16 @@ export const initializeJitsi = async (
     hideConferenceSubject: true,
     // Hides the conference timer.
     hideConferenceTimer: true,
-    prejoinPageEnabled: false,
+    prejoinConfig: {
+      enabled: false,
+    },
     resolution: 720,
     toolbarButtons,
     ...live.live_info.jitsi.config_overwrite,
   };
 
   if (live.join_mode === JoinMode.FORCED && !isInstructor) {
-    configOverwrite.prejoinPageEnabled = false;
+    configOverwrite.prejoinConfig!.enabled = false;
```

Comment:
To be honest, I don't understand why we set this to false since this is already always false...
```suggestion
    configOverwrite.prejoinConfig = {
        ...(configOverwrite.prejoinConfig || {}),
        enabled: false,
    };
```

Reply:
Nevermind for the question, it can be override

Reply:
The suggestion still applies since assert will raises lint error with eslint